src/kibana/plugins/vis_types/vislib/tile_map.js

```diff
@@ -13,7 +13,8 @@ define(function (require) {
       params: {
         defaults: {
           mapType: 'Scaled Circle Markers',
-          isDesaturated: true
+          isDesaturated: true,
+          addLeafletPopup: true
```

Comment:
Use the standard `vis.params.addTooltip` instead of `vis.params.addLeafletPopup`